packages/react-shopify-app-route-propagator/src/test/hook.test.tsx

```diff
@@ -1,12 +1,18 @@
 import * as React from 'react';
-import {mount} from 'enzyme';
 import {ClientApplication} from '@shopify/app-bridge';
 import {History as AppBridgeHistory} from '@shopify/app-bridge/actions';
+import {mount} from '@shopify/react-testing';
+import {MODAL_IFRAME_NAME} from '../globals';
+import useRoutePropagation from '../hook';
 
-import RoutePropagator, {MODAL_IFRAME_NAME} from '..';
+function MockRoutePropagator({app, location}) {
```

Comment:
Hmm I thought we were still testing the "component version" of the hook when there already was one, rather than creating a mock component for tests?

Reply:
I remember the complete opposite for some reason 😖